### PR TITLE
merge 0.13.1 into master

### PR DESCRIPTION
**Pull request by simonsarginson**
#### Description
Describe your changes here.

#### Release note
REQUIRED: Add a release note to the `##Unreleased` section of CHANGELOG.md. You can find guidance for writing useful release notes [here](../SpatialGDK/Extras/internal-documentation/how-to-write-good-release-notes.md). Documentation changes are exempt from this requirement.

#### Tests
How did you test these changes prior to submitting this pull request?
What automated tests are included in this PR?

STRONGLY SUGGESTED: How can this be verified by QA?

#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?

#### Reminders (IMPORTANT)
If your change relies on a breaking engine change:
* Increment `SPATIAL_ENGINE_VERSION` in `Engine\Source\Runtime\Launch\Resources\SpatialVersion.h` (in the engine fork) as well as `SPATIAL_GDK_VERSION` in `SpatialGDK\Source\SpatialGDK\Public\Utils\EngineVersionCheck.h`. This helps others by providing a more helpful message during compilation to make sure the GDK and the Engine are up to date.

If your change updates `Setup.bat`, `Setup.sh`, core SDK version, any C# tools in `SpatialGDK\Build\Programs\Improbable.Unreal.Scripts`, or hand-written schema in `SpatialGDK\Extras\schema`:
* Increment the number in `RequireSetup`. This will automatically run `Setup.bat` or `Setup.sh` when the GDK is next pulled.

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: SpatialGDK/Extras/schema/debug_metrics.schema [hunk 1]
```diff
@@ -8,9 +8,16 @@ type ModifySettingPayload {
     float setting_value = 2;
 }
 
+type ExecServerCommandPayload {
```
Comment (m-samiec) [inline SpatialGDK/Extras/schema/debug_metrics.schema:11]: Should increment `RequireSetup`

**Review by m-samiec — APPROVED** [review]

**Review by improbable-dmitrii — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -43,9 +42,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+## [`0.13.1`] - 2021-06-02
+
+### Breaking changes:
+- Event tracing has been optimised to reduce overhead when tracing events in general and in particular when events are not sampled. The tracing API has been modified to accommodate these improvements. You will have to modify your project if you use the API.
+
+### Features:
+- Added `SpatialExecServerCmd` console command with one command `StartInsights` to dynamically enable insights capturing.
+  - Format: SpatialExecServerCmd < server > < command > < args >
+  - Example usage: "SpatialExecServerCmd local StartInsights -trace=CustomChannel -tracefile=MyNewTrace"
+- Visual Logger now supports multi-worker environments.
```
Comment (joshuahuburn) [inline CHANGELOG.md:57]: Looks like this is a dupe 
image

Comment (simonsarginson) [reply CHANGELOG.md:57]: Was done in both master and 13.1 so its an honest dupe I guess..

Comment (joshuahuburn) [reply CHANGELOG.md:57]: Yeah fair enough

**Review by joshuahuburn — APPROVED** [review]

**Review by mironec — APPROVED** [review]

**Pull request merged**